Sprinkle of Yeezus/TimePicker.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  TimePicker.swift
+//  Sprinkle of Yeezus
+//
+//  Created by Eric Bates on 6/20/18.
+//  Copyright © 2018 Eric Bates. All rights reserved.
+//
+
+import UIKit
+
+protocol TimePickerDelegate: class {
+    
+    func didUpdatePicker(date: Date)
+}
+
+class TimePicker: UIViewController {
+
+    @IBOutlet weak var timePicker: UIDatePicker!
+    
+    weak var delegate: TimePickerDelegate?
+    
+    @IBAction func getTimeFromPicker(_ sender: UIDatePicker) {
+        delegate?.didUpdatePicker(date: sender.date)
+    }
+    
+    override func viewDidLoad() {
```

Comment:
maybe use viewDidLoad to set the timePicker to the user's current notification time?